src/components/ListingCard/styles.tsx

```diff
@@ -1,32 +1,44 @@
 import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
 
 // the card itself
 export const CardBody = styled.div<{ $selected?: boolean }>`
   display: flex;
   flex-direction: column;
-  border: 1px solid lightgray;
+  border: 2px solid ${COLORS.midGrey};
```

Comment:
Let's go a little crazier, say `3px` (`0.1875rem`)

Reply:
Wondering if there's a reason you think we should do `3px` instead? I think David wants `2px` borders for most components, and this is also currently consistent with the Figma design

Reply:
Oh right, I think it was showing up too thin compared to the Figma. `2px` seems to be what it is. Although on Figma it seems to be "tearful blue"?

Reply:
Oh yeah, it's only "tearful blue" when it's selected, this should be on line 19. Otherwise, the color is `greyLight`